### PR TITLE
Siobhan

### PR DESCRIPTION
**Pull request by SMHall94**
Hi, Leonard!
Looking forward to your comments and feedback!

### CODE REVIEW

File: api/app/users/tests.py [hunk 1]
```diff
@@ -566,7 +566,7 @@ def test_response_and_not_invited(self):
 
         # Assert that request succeeds and that users that responded are returned.
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(len(response.json), 1)
+        self.assertEqual(len(response.json), 0)
```
Comment (lzbotha) [inline api/app/users/tests.py:569]: I think there should be users returned here. We  want to check that users that have responded (but are not invited guests) are returned. And the response should include that they are not invited guests (in the field we added).

File: api/app/users/tests.py [hunk 2]
```diff
@@ -594,9 +594,7 @@ def test_no_response_and_invited(self):
 
         # Assert that request succeeds and no users are returned.
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(response.json, [])
-
-        self.assertTrue(False)
+        self.assertEqual(len(response.json), 1)
```
Comment (lzbotha) [inline api/app/users/tests.py:597]: Ideally we should check the contents of the response as well. At least check the uid of the user that is returned to make sure it's the right one.

Comment (SMHall94) [reply api/app/users/tests.py:597]: Okay, great. So, as I have it here, it's just checking that there is a response, and the length thereof?
When you say the contents, do you mean type? I hope that's the right word - but, do you mean check if it's a list / dictionary?

File: api/app/users/tests.py (hunk 1 above)
Comment (SMHall94) [reply api/app/users/tests.py:569]: How do we specify the response that they are not invited guests?

**Pull request merged**